### DISCUSSION TITLE
2.4.10 - Where is Graphana ?

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#11069] [2.4] [answered]
Hi,

Just installed a standalone instance and there is no graphana. Deprecated ?

Thankx

**Comment by dougburks** [comment, marked as answer]
From https://blog.securityonion.net/2023/08/security-onion-24-has-reached-general.html:

Grafana has been removed and all health metrics can be found in InfluxDB:
https://docs.securityonion.net/en/2.4/influxdb.html
